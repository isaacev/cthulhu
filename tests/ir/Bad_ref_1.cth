fn hello_world() -> () {
  -- empty
}

fn main() -> () {
  hello_wlord();
}
