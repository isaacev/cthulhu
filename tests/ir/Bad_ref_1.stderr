
  UNKNOWN NAME  --------------------------------------------------------------

  There was a reference to a name that is not in the current scope.

    : TEST_DIR/ir/Bad_ref_1.cth
  6 | hello_wlord();
    | ^^^^^^^^^^^

  Here are some variables that are similar:

    hello_world

